mdonatello/mdonatello.py

```diff
@@ -9,3 +9,138 @@
 import base64
 
 
+class MoleculeVisualizer:
```

Comment:
add in a class docstring here that includes
- a description of what this class does
- example(s) of using this class

Reply:
Ah yes, my bad, will do it :)